components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/config/model/graph/js/JsClaims.java

```diff
@@ -478,4 +480,10 @@ protected void setRuntimeClaim(String claimUri, Object claimValue) {
         }
         getContext().addRuntimeClaim(claimUri, String.valueOf(claimValue));
     }
+
+    private boolean isAuthenticatedUserInCurrentTenant() {
+
+        return authenticatedUser != null && Objects.equals(PrivilegedCarbonContext.getThreadLocalCarbonContext()
+                .getTenantDomain(), authenticatedUser.getTenantDomain());
```

Comment:
Will it return true if PrivilegedCarbonContext.getThreadLocalCarbonContext()
                .getTenantDomain() = null and authenticatedUser.getTenantDomain() also null